src/pages/NotFound/NotFound.js

```diff
@@ -0,0 +1,12 @@
+// 404 Page
+
+import React from 'react';
+
+const NotFound = () => (
+  <div>
+    <h2>404 - Page not found</h2>
+    <p>We&apos;re sorry, the page you were looking for cannot be found!</p>
```

Comment:
I would add some styles for this page - maybe centre this and increase the font. 
Personally, I like the 404 of the mui.com - it looks very stylish and laconically. But you can find another example if you would like to.
To add styles, we use Styled Components with "@emotion/styled" library which comes with Material UI. You can find an example on Wallet.js page.